.babelrc

```diff
@@ -4,10 +4,20 @@
     ["env", {
       "targets": {
         "uglify": true,
-        "browsers": ["last 2 versions", "ie 11", "not ie 10"]
+        /* This is the full list of browsers we support, but we are not using it because we want babel-polyfill
+         * to include only polyfills for more modern browsers. 
+         * Older browser polyfills are in preModernBrowserPolyfills.js
+         */
+        /* "browsers": ["Chrome 60", "Firefox 57", "iOS 9", "Edge 14", "ChromeAndroid 64", "Safari 10", "ie 11"] */
+        /*
+         * This is the list of browers we support that all also have support for the nomodule script
+         * attribute, meaning we can make them ignore a file that includes polyfills for older browsers
+         */
+        "browsers": ["Chrome 61", "Firefox 60", "iOS 11", "Edge 16", "ChromeAndroid 67", "Safari 11"]
```

Comment:
I think this will become easier in Babel 7, where you can specifically target browsers that support ES modules.